Userland/Libraries/LibGfx/CMakeLists.txt

```diff
@@ -78,3 +77,26 @@ set(SOURCES
 
 serenity_lib(LibGfx gfx)
 target_link_libraries(LibGfx PRIVATE LibCompress LibCore LibCrypto LibFileSystem LibTextCodec LibIPC LibUnicode)
+
+set(generated_sources TIFFMetadata.h TIFFTagHandler.cpp)
+list(TRANSFORM generated_sources PREPEND "ImageFormats/")
+
+find_package(Python COMPONENTS Interpreter REQUIRED)
+
+add_custom_command(
+        OUTPUT  ${generated_sources}
+        COMMAND "${CMAKE_COMMAND}" -E make_directory "tmp"
+        COMMAND ${Python_EXECUTABLE} "${CMAKE_CURRENT_SOURCE_DIR}/tiff_generator.py" -o "${CMAKE_CURRENT_BINARY_DIR}/tmp"
+        COMMAND "${CMAKE_COMMAND}" -E copy_if_different tmp/TIFFMetadata.h "ImageFormats/TIFFMetadata.h"
+        COMMAND "${CMAKE_COMMAND}" -E copy_if_different tmp/TIFFTagHandler.cpp "ImageFormats/TIFFTagHandler.cpp"
+        COMMAND "${CMAKE_COMMAND}" -E remove_directory "${CMAKE_CURRENT_BINARY_DIR}/tmp"
+        VERBATIM
+)
+target_sources(LibGfx PRIVATE ${generated_sources})
+add_custom_target(generate_tiff_handler DEPENDS ${generated_sources})
+add_dependencies(all_generated generate_tiff_handler)
+add_dependencies(LibGfx generate_tiff_handler)
+
+list(FILTER generated_sources INCLUDE REGEX "\.h$")
+list(TRANSFORM generated_sources PREPEND "${CMAKE_CURRENT_BINARY_DIR}/")
+install(FILES ${generated_sources} DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/LibGfx/ImageFormats")
```

Comment:
This isn't correct for the serenity use case. We normally install all source files, (I think? maybe we are missing some generated file installs....).

Reply:
To clarify, you want me to remove the filter so it installs both `.h` and `.cpp`, isn't it?
FYI: I took that from here:
https://github.com/SerenityOS/serenity/blob/778f10fb262e4330f15014488efff69e92b3ded9/Meta/CMake/libweb_generators.cmake#L193-L224